### DISCUSSION TITLE
basic vim plugin: `:FZF [options]` not working

### DISCUSSION THREAD

**Discussion by camoz** [junegunn/fzf#3919] [General]
Hi,

I have `fzf 0.53.0 (c4a9ccd6)` and Neovim 0.10 installed. The command `:FZF` works fine, but `:FZF --reverse` or `:FZF --height=3` does not, i.e. it seems that I cannot supply options to the basic `:FZF` command, which should be supported acc. to https://github.com/junegunn/fzf/blob/master/README-VIM.md#summary.

Am I missing something? Does it work for other people?

Thanks :)

**Comment by junegunn** [comment]
`:FZF --reverse` works for me. `--height` option, however, is not supposed to work due to the limitation of Neovim, where fzf has to start in a separate terminal buffer rather than below the editor. `:call fzf#run({'down': 3})` is the closest thing you can do where the Vim plugin of fzf splits the window and opens fzf inside the terminal buffer on the split window.

Having said that, `:FZF options` is quite limited, and if you're looking to implement custom commands and functions using fzf, you should use `fzf#run` (and almost always with `fzf#wrap`) e.g. `call fzf#run(fzf#wrap({'options': '--no-color'}))`

**Comment by camoz** [comment]
I see, thanks for the answer! (I actually had enabled `--reverse` via `FZF_DEFAULT_OPTS` which is why I thought it wasn't working, too. I guess I could have tested more options...)
